src/pages/discovery/loadingAnimation/loadingAnimation.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./loadingAnimation.scss";
+import loadingAnimation from "../../../assets/loadingAnimationAssets/loadingAnimation.svg";
+
+/**
+ * Renders a centered loading animation when active. To use, initialize with 'loading' prop, and set to true for display,
```

Comment:
Couple of things here:

1. This looks and works great. Good job!
2. I imagine we may want to use this in other places as the site grows. This component should not live in the Discovery directory. This should be renamed, generalized where necessary, and relocated to the components directory to be used site wide as needed.
3. Update this boolean values, ternary logic, and variable names so that it is more readable and matches the comments discovery.js
4. Update the component to use PropTypes so that we can implement type checking. Most, if not all of the components have been updated to use this pattern. You can use any of them as an example.